COMPLETE LIST OF SCRIPTS.vbs

```diff
@@ -2249,6 +2249,7 @@ script_array(script_num).tags                   = array("BZ", "Monthly Tasks", "
 script_array(script_num).dlg_keys               = array("Up", "Ex", "Ev")
 script_array(script_num).subcategory            = array("")
 script_array(script_num).release_date           = #10/01/2000#
+script_array(script_num).retirement_date        = #01/23/2024#
```

Comment:
Are we moving inactive transfer to the retired scripts?

Reply:
The BULK - INACTIVE TRANSFER is being retired temporarily while the 'excluded' baskets and the entire need of transferring cases is re-evaluated.

Reply:
Nope, this will take a minute I'm sure.